src/core/a-entity.js

```diff
@@ -258,7 +258,11 @@ var proto = Object.create(ANode.prototype, {
    */
   remove: {
     value: function (el) {
-      this.object3D.remove(el.object3D);
+      if (el) {
+        this.object3D.remove(el.object3D);
+      } else {
+        this.object3D.remove(...this.object3D.children);
```

Comment:
Syntax not correct:

`this.object3d.remove(...this.object3D.children);`

Reply:
We needed to add `this.parentNode.removeChild(this)` too, otherwise the nodes weren't being deleted from the DOM.

Reply:
Notice the three dots. That's not correct JS syntax: ` this.object3D.remove(...this.object3D.children);`

Reply:
Have removed the spread operator. Using a while loop instead to remove each child Object3D.